TrenDAP/Model/DataSource.cs

```diff
@@ -23,23 +23,43 @@
 
 using Gemstone.Data;
 using Gemstone.Data.Model;
+using GSF.Data.Model;
+using GSF.Security.Model;
 using Microsoft.AspNetCore.Components;
 using Microsoft.Extensions.Configuration;
+using openXDA.APIAuthentication;
+using System;
 using System.Security;
 using TrenDAP.Controllers;
+using PrimaryKeyAttribute = Gemstone.Data.Model.PrimaryKeyAttribute;
+using UseEscapedNameAttribute = Gemstone.Data.Model.UseEscapedNameAttribute;
 
 namespace TrenDAP.Model
 {
-
+    // Todo: ParentKey & Custom view not respected by custom model controller implementation
```

Comment:
Why are we adding these if they're not supported?

Reply:
So this branch is base to some other ones. Intention being that they would be merged as I worked, but it took longer than expected for the one before this to get merged, so some things change down the line. This issue gets resolved here
 https://github.com/GridProtectionAlliance/TrenDAP/commit/2778ad1094b5b154ee68e3a38bb40a85a7850431
 
 Apologies for the confusion.